src/bun.js/bindings/BunString.cpp

```diff
@@ -69,13 +69,7 @@ extern "C" BunString BunString__createAtom(const char* bytes, size_t length)
 {
     ASSERT(simdutf::validate_ascii(bytes, length));
     auto atom = tryMakeAtomString(String(StringImpl::createWithoutCopying({ bytes, length })));
```

Comment:
this isn't part of this pr, but we aren't checking for `atom.isNull()` here. should we change this to `makeAtomString`?